src/extension.ts

```diff
@@ -560,5 +513,72 @@ function startKubeProxy(): Promise<string> {
 				reject(new Error(data.toString()));
 			});
 		}
-	}); 
+	});
 }
+
+function determinePlatform(): void {
+	const osType = platform();
+	if (osType === 'win32') {
+		curlCommand = 'cmd';
+		curlOption = '/c';
+	}
+}
+
+function applyStatusBarSettings(): void {
+	// process the workspace setting indicating whether we should use the proxy or CLI
```

Comment:
the comment is unrelated to the code

Reply:
that is nothing I added. I am just extracting to functions.

Reply:
I would prefer to remove the 3 comment lines that are even more useless as the refactoring is making the code clearer, and allows to easily spot that one of them is incorrect

Reply:
done